### DISCUSSION TITLE
✅ July 2024 (v0.4)

### DISCUSSION THREAD

**Discussion by flanakin** [microsoft/finops-toolkit#61] [Releases]
### 🆕 New in July 2024
- 📗 [FinOps guide](https://microsoft.github.io/finops-toolkit/#-learning-resources) updated to FinOps Framework 2024 and FOCUS 1.0.
- 🏦 [FinOps hubs](https://aka.ms/finops/hubs) support for FOCUS 1.0, managed exports, and cross-tenant exports.
- 📊 [Power BI reports](https://aka.ms/ftk/pbi) were all updated to FOCUS 1.0.
- 🌐 [Open data files](https://microsoft.github.io/finops-toolkit/data) were all updated.
- 🔍 [Azure optimization engine](https://microsoft.github.io/finops-toolkit/optimization-engine) - an extensible solution for custom optimization recommendations.

### ℹ️ Updated
- 📊 [Power BI reports](https://aka.ms/ftk/pbi) had many fixes and improvements across all reports!
- 📒 [Cost optimization workbook](https://microsoft.github.io/finops-toolkit/optimization-workbook) updated to include the latest changes from Azure Advisor.
- 📒 [Governance workbook](https://microsoft.github.io/finops-toolkit/governance-workbook) had a few minor updates.
- 🖥️ [FinOps toolkit PowerShell commands](https://aka.ms/ftk/ps) were updated to support FOCUS 1.0 exports.

<br>

📋 Milestone tasks: [July 2024 (v0.4)](https://github.com/microsoft/finops-toolkit/milestone/4)
🌱 Working branches: [exports](https://github.com/microsoft/cloud-hubs/tree/features/exports)

---
### 📥 [Download v0.4](/microsoft/finops-toolkit/releases/v0.4) &nbsp; &nbsp; &nbsp; 📜 [Changelog](https://aka.ms/ftk/changes#-v04) &nbsp; &nbsp; &nbsp; 🖥️ [PowerShell](https://aka.ms/ftk/ps/gallery) &nbsp; &nbsp; &nbsp; 📗 [Learn more](https://aka.ms/finops/toolkit) &nbsp; &nbsp; &nbsp; ❤️ [Give feedback](https://aka.ms/ftk/feedback)

**Comment by boomer4d** [comment]
I'd love to see CSP for Azure Gov subscription support  but recognize that Microsoft has to make changes to the platform for that to happen. Currently the ability to export usage is not supported in this subscription type which renders most of the toolkit useless to those of us mired in CSP for Azure Gov subscription cost management hell.

**Reply by flanakin** [reply]
This is currently blocked by Cost Management. Adding this without Cost Management support would be a fairly big effort. Doable, but tough with limited resources. I would suggest escalating this with your Microsoft account team, if you have one. I suppose another option is to create an issue in our repo and if it gets enough votes, I can try to use that to nudge the right teams to make the move. Either way, it's not a quick option 😔

**Comment by Mabdelhalim79** [comment]
Hi .. I am using Cost summary PBI for my reports .. We had a challenge in reporting tags .. I heard it is resolved in v0.4 but, I still dont see tags tab in the new Cost summary PBI report .. or we need to configure that ?
